src/ASM/ASMbase.h

```diff
@@ -822,6 +824,7 @@ class ASMbase
   unsigned char nLag;   //!< Number of Lagrange multipliers per node
   size_t        nel;    //!< Number of regular elements in this patch
   size_t        nnod;   //!< Number of regular nodes in this patch
+  bool          partitioned = false; //!< If true model is partitioned
```

Comment:
Since the other members are initialized by the constructor, so should this one, for consistency

Reply:
done.